pkg/testing/helm/helm_app.go

```diff
@@ -50,9 +51,10 @@ func (h *HelmApp) AddRepo(name, url string) error {
 	return h.client.AddRepo(context.Background(), name, url)
 }
 
-func (h *HelmApp) Install() error {
-	ctx := context.Background()
-	return h.client.Install(ctx, h.chart, "", h.name, h.namespace, h.helmValues, true)
+func (h *HelmApp) Install() (string, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
```

Comment:
is this timeout required?

Reply:
Not required. This got commited by mistake.
Removed.